src/LibraryManager/Manifest.cs

```diff
@@ -102,11 +106,21 @@ internal static Manifest FromJson(string json, IDependencies dependencies)
 
                 manifest._hostInteraction = dependencies.GetHostInteractions();
 
-                return manifest;
+                return (manifest, null);
             }
-            catch (Exception)
+            catch (Exception e)
             {
-                return null;
+                string fullDiagnostics = e.ToString() + Environment.NewLine +
+                    "---" + Environment.NewLine +
+                    Thread.CurrentThread.Name + Environment.NewLine +
+                    "---" + Environment.NewLine +
+                    Environment.StackTrace + Environment.NewLine +
```

Comment:
Is this stack trace useful?  Doesn't `e.ToString()` contain a stack trace of where the exception was thrown (which would include this frame).